### DISCUSSION TITLE
Issue with libstdc++.so.6 GLIBC version - `GLIBCXX_3.4.26' not found

### DISCUSSION THREAD

**Discussion by agb2008** [neutralinojs/neutralinojs#416] [General]
I've been trying to test **neutralinojs** on my Linux workstation (with **CentOS 7 x64 clone OS**) but facing following issue - when I tried to generate first binaries with just default demo project:

">neu run

------- Notice --------

Neutralino v2 is ready for Linux/Windows. You can try v2.0.0 pre-release.
Please notice that the macOS version is not yet supporting v2 implementaton (still in v1.5.0).
If you are interested, you can help <3 us to implement v2-spec for macOS.
Checkout more info: https://github.com/neutralinojs/neutralinojs

**./myapp-linux: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by ./myapp-linux)**

Application was terminated."

It seems that some of the modules in **neutralinojs** package got dynamic link with that **libstdc++.so.6** library
and, as a result, on all other linux distributions, that use older version of **libstdc++.so.6** (in my case it's **3.4.25**)
execution of generated application would fail.

Possible solution would be to use static link for this library by passing **-static-libstdc++** to **g++** when linking
modules that would be used on other **Linux OS** versions with different version of **libstdc++**...

**Comment by prakhar-pal** [comment]
I faced the same issue on Ubuntu 18.04. 

But when I compiled it from neutralinojs/core-linux, it compiled successfully. I was able to use the newly compiled file instead of the one that comes in v2 minimal app and it worked fine.
So it does seem like the issue is really how it's compiled. Can't we move that compilation step to user's computer itself? Like node-sass bindings need to be built for nodejs? This will remove need of all the platform binaries as well.

**Comment by ghost** [comment]
I've hit a similar error on centos 7. The problem is centos 7 only comes with GLIBC 2.17 and GLIBCXX_3.4.19

So does neurtralinojs support centos 7?
